test/cmd/explain.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
+trap os::test::junit::reconcile_output EXIT
+
+# Cleanup cluster resources created by this test
+(
+  set +e
```

Comment:
Why `set +e` if we it can be written as:
```bash
oc delete all,templates --all || :
```
?

Reply:
I followed the pattern seen in other tests: https://github.com/openshift/origin/blob/master/test/cmd/builds.sh#L7